container-metrics.html.md.erb

```diff
@@ -126,3 +130,26 @@ Showing CPU usage against entitlement for app dora-example in org example-org /
 ​
 </pre>
 
+### <a id="cf-app"></a> The CPU overentitlement plugin
+
+The plugin shows when your app has been throttled which helps you forecast when to upscale.
+
+You can check all the so called bad applications in your organization by using the CPU overentitlement plugin and the `cf over-entitlement-instances` command.
+
+1. Install the CPU Overentitlement Plugin from [cpu-entitlement-plugin](https://github.com/cloudfoundry/cpu-entitlement-plugin) in GitHub.
```

Comment:
Is this the correct link? Is the overentitlement plugin the same as the entitlement plugin?

Reply:
The CPU overentitlement plugin is downloaded from the repository of the CPU entitlement plugin and that's why the link is the same.